### ISSUE TITLE
Long question (original)

### ISSUE THREAD

**Issue by vickytnz** [vickytnz/accessibility-examples#2]
The issue with this one for screen-reader users is the lack of legend (or accessible name) on the radio group. However as a missing legend (as you currently have it marked up) will be flagged in some automated tests this is sometimes “fixed” (and the visual design retained) by using a visually-hidden legend. But this leaves screen magnification, mobile and (as you say) those with memory conditions having to scroll back up to see the question, so we advise the use of the complex question pattern (which is your fixed version).

Might be worth a direct link to the complex question pattern on [http://GOV.UK](https://t.co/G73rGwTpwY)?
https://design-system.service.gov.uk/patterns/question-pages/#asking-complex-questions-without-using-hint-text.

You are also introducing a link in the fixed page which wasn’t present on the “before” version - I’d suggest moving this to the “Short text area” example as it fits that example better and will allow this one to focus on the structure more.

Suggest renaming this to “Complex question”.

Reported by @adamliptrot-oc
